tiles/src/main/java/com/protomaps/basemap/layers/Pois.java

```diff
@@ -3,85 +3,308 @@
 
   @Override
   public String name() {
     return "pois";
   }
 
+  private static final double WORLD_AREA_FOR_70K_SQUARE_METERS =
+          Math.pow(GeoUtils.metersToPixelAtEquator(0, Math.sqrt(70_000)) / 256d, 2);
+  private static final double LOG2 = Math.log(2);
+
   @Override
   public void processFeature(SourceFeature sf, FeatureCollector features) {
-    if (!sf.canBeLine() && (sf.isPoint() || sf.canBePolygon()) && (sf.hasTag("amenity") ||
-      sf.hasTag("shop") ||
-      sf.hasTag("tourism") ||
-      sf.hasTag("leisure") ||
-      sf.hasTag("aeroway", "aerodrome") ||
-      sf.hasTag("railway", "station"))) {
+    if ( (sf.isPoint() || sf.canBePolygon()) && (
+            sf.hasTag("aeroway", "aerodrome") ||
+            sf.hasTag("amenity") ||
+            sf.hasTag("boundary", "national_park", "protected_area") ||
+            sf.hasTag("craft") ||
+            sf.hasTag("historic") ||
+            sf.hasTag("landuse", "cemetery", "recreation_ground", "winter_sports", "quarry", "park", "forest", "military") ||
+            sf.hasTag("leisure") ||
+            sf.hasTag("railway", "station") ||
+            sf.hasTag("shop") ||
+            sf.hasTag("tourism") ))
+    {
       String kind = "other";
+      String kind_detail = "";
+      Integer min_zoom = 15;
+
       if (sf.hasTag("aeroway", "aerodrome")) {
         kind = sf.getString("aeroway");
-      } else if (sf.hasTag("amenity", "cafe", "college", "hospital", "library", "post_office", "school", "townhall")) {
+        min_zoom = 13;
+
+        // Emphasize large international airports earlier
+        if( kind == "aerodrome" &&  sf.hasTag("iata")) {
+          min_zoom -= 2;
+        }
+
+        if( sf.hasTag("aerodrome") ) {
+          kind_detail = sf.getString("aerodrome");
+        }
+      } else if (sf.hasTag("amenity",  "university","college")) {
+        kind = sf.getString("amenity");
+        min_zoom = 11;
+      } else if (sf.hasTag("amenity", "hospital")) {
+        kind = sf.getString("amenity");
+        min_zoom = 12;
+      } else if (sf.hasTag("amenity", "library", "post_office", "school", "townhall")) {
         kind = sf.getString("amenity");
+        min_zoom = 13;
+      } else if (sf.hasTag("amenity", "cafe")) {
+        kind = sf.getString("amenity");
+        min_zoom = 15;
       } else if (sf.hasTag("landuse", "cemetery")) {
         kind = sf.getString("landuse");
+        min_zoom = 14;
       } else if (sf.hasTag("leisure", "golf_course", "marina", "park", "stadium")) {
         kind = sf.getString("leisure");
-      } else if (sf.hasTag("leisure")) {
-        // This is dubious but existing behavior
-        kind = "park";
+        min_zoom = 13;
       } else if (sf.hasTag("shop", "grocery", "supermarket")) {
         kind = sf.getString("shop");
+        min_zoom = 14;
       } else if (sf.hasTag("tourism", "attraction", "camp_site", "hotel")) {
         kind = sf.getString("tourism");
+        min_zoom = 15;
+      } else {
```

Comment:
If it's not in the allowlist, still let it thru but with a deferred min zoom of 15.